app/controllers/application_controller.rb

```diff
@@ -143,6 +145,14 @@ def set_locale!
     I18n.locale = Organization.current.locale
   end
 
+  def set_time_zone!
+    Time.zone = Organization.current.time_zone
+  end
+
+  def unset_time_zone!
```

Comment:
Why do we need to manually unset the time zone? I think we are not doing that with `current_user` and `Organization.current`, are we?

Reply:
> and `Organization.current`, are we?

we do, actually!
https://github.com/mumuki/mumuki-laboratory/blob/a41ffc14f7b45b5ed7c0578176f3b4caaacb9a74/app/controllers/application_controller.rb#L33
https://github.com/mumuki/mumuki-laboratory/blob/a41ffc14f7b45b5ed7c0578176f3b4caaacb9a74/app/controllers/application_controller.rb#L160-L162

as for current_user, not 100% sure, but I thought it doesn't survive the request as it's not saved as a thread variable?

Reply:
> as for current_user, not 100% sure, but I thought it doesn't survive the request as it's not saved as a thread variable?

https://stackoverflow.com/a/53671186